manifests/k/kvakulo/switcheroo/0.9.2/kvakulo.switcheroo.locale.en-US.yaml

```
@@ -0,0 +1,13 @@
+# Created using wingetcreate 1.1.2.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.2.0.schema.json
+
+PackageIdentifier: kvakulo.switcheroo
+PackageVersion: 0.9.2
+PackageLocale: en-US
+Publisher: kvakulo

```

```suggestion
Publisher: Regin Larsen
```
This must match Control Panel.